src/FMBot.Bot/Models/MusicBot/JockieMusicBot.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using Discord;
+using Discord.WebSocket;
+
+namespace FMBot.Bot.Models.MusicBot;
+
+internal class JockieMusicBot : MusicBot
+{
+    private const string StartedPlaying = " ​ Started playing ";
+    public JockieMusicBot() : base("Jockie Music", true)
+    {
+    }
+
+    public override bool ShouldIgnoreMessage(IUserMessage msg)
+    {
+        if (msg.Embeds.Count != 1)
+        {
+            return true;
+        }
+        var description = msg.Embeds.First().Description;
+        return string.IsNullOrEmpty(description) || !description.Contains(StartedPlaying);
+    }
+
+    /**
+     * Example: :spotify: ​ Started playing Giants by Lights
+     */
+    public override string GetTrackQuery(IUserMessage msg)
+    {
+        var description = msg.Embeds.First().Description;
+        var songByArtist = description[description.IndexOf(StartedPlaying, StringComparison.Ordinal)..];
+        return songByArtist;
```

Comment:
This is a minor codestyle thing, but you can just return it directly, no need to define it first